src/app/components/groups/groups.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+  <div class="title">
+    <h2>Groups around your area and interest</h2>
+  </div>
+
+  <div class="button-container">
+    <button class="btn btn-primary" routerLink="/new-group">
+      Start a new Group
+    </button>
+  </div>
+
+  <div class="groups-container">
+    <div><h5>My Groups</h5></div>
+
+    <!-- If My groups available -->
+    <div *ngIf="isMyGroupsAvailable">
```

Comment:
Try single line https://stackoverflow.com/questions/43812770/ngif-else-if-in-template

Reply:
Thanks for the tip